packages/mdx/types/index.d.ts

```diff
@@ -11,35 +11,35 @@ declare namespace mdx {
     filepath?: string
 
     /**
-     * skip the addition of 'export default' statement when serializing
+     * Skip the addition of 'export default' statement when serializing
```

Comment:
Is this linting related?
I'm not seeing a related lint rule in the configuration

Reply:
https://eslint.org/docs/rules/capitalized-comments

Maybe xo enables it, I'm not sure.